packages/ui-react/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import classNames from 'classnames';
+import { IS_DEVELOPMENT_BUILD } from '@jwp/ott-common/src/utils/common';
+
+import MarkdownComponent from '../MarkdownComponent/MarkdownComponent';
+
+import styles from './Footer.module.scss';
+
+type Props = {
+  text: string;
+};
+
+const Footer: React.FC<Props> = ({ text }) => {
+  const chunks = text.split('|');
+
+  return (
+    // The extra style below is just to fix the footer on mobile when the dev selector is shown
+    <footer className={classNames(styles.footer, { [styles.testFixMargin]: IS_DEVELOPMENT_BUILD })}>
+      {(() => {
+        const footerContent = chunks.map((value, index) => (
+          <MarkdownComponent key={index} markdownString={value} inline tag={chunks.length > 1 ? 'li' : 'div'} />
+        ));
+
+        if (chunks.length > 1) {
+          return <ul className={styles.list}>{footerContent}</ul>;
+        } else {
+          return footerContent;
+        }
+      })()}
+    </footer>
+  );
+};
```

Comment:
I have doubts about using a list here. Could you use the same logic that we did for the VideoMetaTags component?

Is using a list better than separating the elements? If so, you can ignore this suggestion. Although I would try preventing the self-invoking function in JSX.

```suggestion
const Footer: React.FC<Props> = ({ text }) => {
  const chunks = text.split('|').map(chuck => chuck.trim());

  return (
    // The extra style below is just to fix the footer on mobile when the dev selector is shown
    <footer className={classNames(styles.footer, { [styles.testFixMargin]: IS_DEVELOPMENT_BUILD })}>
      {chunks.map((value, index) => (
        <>
          <MarkdownComponent key={value} markdownString={value} inline tag="span" />
          {chunks.length > 1 && index !== chunks.length - 1 ? (<span className="sr-only"> | </span>) : null}
        </>
      ))}
    </footer>
  );
};
```

Reply:
Sorry @langemike, I see that lists are pretty common in footer navigations. Too bad we can't use a `nav` or `role="navigation"` because it's no guarantee all chunks are links...

Reply:
```suggestion
const Footer: React.FC<Props> = ({ text }) => {
  const chunks = text.split('|').map(chuck => chuck.trim());
  const footerContent = chunks.map((value, index) => (
    <MarkdownComponent key={index} markdownString={value} inline tag={chunks.length > 1 ? 'li' : 'div'} />
  ));
  const content = chunks.length > 1 ? <ul className={styles.list}>{footerContent}</ul> : footerContent;

  return (
    // The extra style below is just to fix the footer on mobile when the dev selector is shown
    <footer className={classNames(styles.footer, { [styles.testFixMargin]: IS_DEVELOPMENT_BUILD })}>
      {content}
    </footer>
  );
};
```

Reply:
As discussed: we keep the list and apply trim to the values and removed the self-invoking function. Also modified the CSS a little bit to make the styling correct (due to missing spaces in the list-items).

I updated the code